.github/workflows/java.yml

```diff
@@ -2,9 +2,9 @@ name: Gradle CI/CD
 
 on:
   push:
-    branches: [ actions ]
+    branches: [ main ]
   pull_request:
-    branches: [ actions ]
+    branches: [ main ]
```

Comment:
ta nazwa mówi o base branch (to, co tu jest w dropdown):
![image](https://github.com/yarpo/wolo-app-api/assets/5647337/67ea44da-61cc-4719-b378-8baa6cfc549c)

Czyli do czego porównujemy diffa oraz do czego chcemy potem zmerge'ować wyniki pracy.